src/index.ts

```diff
@@ -1,4 +1,4 @@
-export { reportWebVitals } from './webVitals';
+export { reportWebVitals, reportWebVitalsWithPath } from './webVitals';
```

Comment:
Does it make sense to export only one of these?
If I understood the code correctly, calling `reportWebVitals` does the same as calling `reportWebVitalsWithPath` without giving a second argument 🤔

Reply:
I did that first but got typescript errors because we export a function that doesn't cohere with the signature Next.js expects. This is the safer way ¯\\_(ツ)_/¯